src/Surfnet/ServiceProviderDashboard/Domain/Entity/Service.php

```diff
@@ -63,6 +65,13 @@ class Service
     #[ORM\GeneratedValue(strategy: 'AUTO')]
     private $id;
 
+    /**
+     * @var DateTime $created
+     */
+    #[Gedmo\Timestampable(on: 'create')]
+    #[ORM\Column(type: 'datetime')]
+    private DateTime $created;
```

Comment:
Isn't it better to rename this to `createdAt`

Reply:
It's a better name, thanks for the suggestion.